lambdas/functions/status/status.go

```diff
@@ -91,3 +96,41 @@ func GetClusterStatus(ctx context.Context, stateTableName, tableHashKey, stateKe
 
 	return
 }
+
+func GetWekaStatus(ctx context.Context, usernameId string, passwordId string, clusterName string) (protocol.WekaStatus, error) {
+	logger := logging.LoggerFromCtx(ctx)
+	logger.Info().Msgf("fetching weka status from %s...", clusterName)
+
+	credentials, err := common.GetUsernameAndPassword(usernameId, passwordId)
```

Comment:
the base branch should be `dev` and you should use `GetWekaDeploymentCredentials`